src/pages/shikshagraha-repository/listing/Header.jsx

```
@@ -154,23 +136,13 @@ export default function Header({ isHeroSection = true }) {
-                  href={`${BASE_URL}/story-archive`}
-                  className="nav-link w-nav-link"
-                >
+                <a href={`${BASE_URL}/story-archive`} className="nav-link w-nav-link">
                   Stories of Impact
                 </a>
-                <a
-                  href="https://docs.google.com/forms/d/e/1FAIpQLSfSX2bzdJzPBOlstfGg7vWqPFaS5weLnPpwIieR1DBdRgepPg/viewform"
-                  target="_blank"
-                  className="nav-link w-nav-link"
-                >
+                <a href="https://docs.google.com/forms/d/e/1FAIpQLSfSX2bzdJzPBOlstfGg7vWqPFaS5weLnPpwIieR1DBdRgepPg/viewform" target="_blank" className="nav-link w-nav-link">
```

Biome (2.1.2)

[error] Avoid using target="_blank" without rel="noopener" or rel="noreferrer".

Opening external links in new tabs without rel="noopener" is a security risk. See the explanation for more details.
Safe fix: Add the rel="noopener" attribute.


(lint/security/noBlankTarget)